web/api/v1/api.go

```diff
@@ -838,6 +839,7 @@ func (api *API) metricMetadata(r *http.Request) apiFuncResult {
 		}
 	}
 
+	// Put the elements from the pseudo-set into a slice for marshaling.
```

Comment:
Vertical whitespace around the inner for could also be reduced.

Reply:
Done.